src/utils/git.ts

```diff
@@ -4,6 +4,14 @@ import { getConfigFileDir } from '../config';
 import { ConfigurationError } from './errors';
 import { logger } from '../logger';
 
+export interface GitChange {
+  hash: string;
+  message: string;
+  pr: string | null;
+}
+
+export const PRExtractor = /(?<=\(#)\d+(?=\)$)/;
```

Comment:
TIL.

We should add to the README the expected format of the commit messages in `auto` mode, to include the PR ID.

Reply:
I don't think that's a good idea as GitHub does this automatically and I don't see a reason for anyone to modify that (unless they are using Phabricator etc. which disqualifies them for this anyway)

Reply:
Would be good to comment on the GitHub behavior we're depending on here.

Reply:
Added comments here: https://github.com/getsentry/craft/pull/289